changelogs/fragments/2830-npm-version-update.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - "npm - When the version property is used in the npm plugin,
+    the comparison of installed vs missing will be used as name@version instead of just name,
+    allowing for version specific updates.
+    (https://github.com/ansible-collections/community.general/issues/2021)."
```

Comment:
```suggestion
  - "npm - When the ``version`` option is used the comparison of installed vs missing will 
    use name@version instead of just name, allowing for version specific updates 
    (https://github.com/ansible-collections/community.general/issues/2021)."
```